turbinia/workers/binary_extractor.py

```diff
@@ -28,6 +28,29 @@
 class BinaryExtractorTask(TurbiniaTask):
   """Extract binaries out of evidence and provide JSON file with hashes."""
 
+  def check_extraction(self, binary_extraction_dir):
+    """Checks counts for extracted binaries and hashes.
+
+    Args:
+      binary_extraction_dir (str): path to folder with extracted binaries.
+
+    Returns:
+      Tuple(
+        binary_cnt(int): Number of extracted binaries.
+        hash_cnt(int): Number of extracted hashes.
+      )
+    """
+    json_path = os.path.join(binary_extraction_dir, 'hashes.json')
+
+    with open(json_path) as json_file:
+      hashes = json.load(json_file)
+
+    binary_cnt = sum(
+        len(files) for _, _, files in os.walk(binary_extraction_dir)) - 1
+    hash_cnt = len(hashes)
+
+    return (binary_cnt, hash_cnt)
+
   def run(self, evidence, result):
```

Comment:
Sorry, I should have asked this in the last review, but could you add a quick test of the run method too?  It should be fairly easy and can be similar to this:
https://github.com/google/turbinia/blob/7d185da0af88460497eda43aa8f9e2c23ae097d3/turbinia/workers/bulk_extractor_test.py#L48

Reply:
Done, I refactored the tests a bit.